examples/systemd/sysconfig.node_exporter

```diff
@@ -0,0 +1 @@
+OPTIONS="-collector.textfile.directory /var/lib/node_exporter/textfile"
```

Comment:
Well, anything there will be OK for LSB, so it's up to you. Do you prefer textfile_collector?